.github/workflows/ci.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           Remove-Item $log_path
           Unregister-OpenTelemetryForCurrentSession
           dotnet help
-          If (Test-Path $log_path) { throw "Log file exist. Instrumentation uninstall failed." }
+          If (Test-Path $log_path) { throw "Log file exists. Instrumentation uninstall failed." }
```

Comment:
Maybe we should also test the uninstall function?

Reply:
testing if the core files were removed?

Reply:
yup I am just not sure if `Remove-Item -LiteralPath $InstallDir -Force -Recurse` works 😅

Reply:
Here is the reason of my doubts https://github.com/signalfx/splunk-otel-dotnet/pull/41/commits/aa317bdbc1f48eb95a9db5bf5095ae3f75ba111f

Reply:
removing dir vs contents, but still no issues, can add.